docs/components/iframe.md

```diff
@@ -0,0 +1,50 @@
+# matestack core component: Iframe
+
+Show [specs](/spec/usage/components/iframe_spec.rb)
+
+The HTML iframe tag implemented in ruby.
+
+## Parameters
+
+This component can take 3 configuration params and either yield content or display what gets passed to the `text` configuration param.
+
+#### # id (optional)
+Expects a string with all ids the `iframe` should have.
+
+#### # class (optional)
+Expects a string with all classes the `iframe` should have for styling purpose.
+
+
+#### # src
```

Comment:
I suspect the `src` does not work since it is not included in the `iframe.rb` file, as the other missing attributes. Therefore, the tests should fail, too?